demos/common/mqtt_demo_helpers/mqtt_demo_helpers.c

```diff
@@ -881,6 +881,30 @@ BaseType_t PublishToTopic( MQTTContext_t * pxMqttContext,
 }
 /*-----------------------------------------------------------*/
 
+BaseType_t ProcessLoop( MQTTContext_t * pxMqttContext,
+                        uint32_t ulTimeoutMs )
+{
+    BaseType_t xReturnStatus = pdFAIL;
+    MQTTStatus_t xMQTTStatus = MQTTSuccess;
+
+    xMQTTStatus = MQTT_ProcessLoop( pxMqttContext, ulTimeoutMs );
+
+    if( xMQTTStatus != MQTTSuccess )
+    {
+        LogWarn( ( "MQTT_ProcessLoop returned with status = %u.",
+                   xMQTTStatus ) );
```

Comment:
I believe we have a function for this
```suggestion
        LogWarn( ( "MQTT_ProcessLoop returned with status = %s.",
                   MQTT_Status_strerror( xMQTTStatus ) ) );
```